tasks/shell_crmsh/cluster-destroy-crm.yml

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Set fact list of configuration files
+  ansible.builtin.set_fact:
+    __ha_cluster_config_files:
```

Comment:
since you don't use __ha_cluster_config_files anywhere else except for the `stat` you can define this variable as a `vars` on the `stat` task (or simply eliminate the variable altogether, unless you intend to add more files or use this elsewhere)

Reply:
In general, it is better to avoid using `set_fact` as it "pollutes" the global namespace, while using `vars` can be scoped to a context

Reply:
Makes sense to me, cannot see `__ha_cluster_config_files` private variable used elsewhere and the files list is unlikely to change in future PCMK versions or OS versions?

Proposal below to use directly in the loop is most condensed method, or the alternative to add a task-level `vars:` underneath the loop.

Reply:
You are correct, i added it just in case there will be more files in future. I am changing it as mentioned below.